lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -810,3 +823,122 @@ PYI_POST_OFFICE:
     end:
       targetJourney: INELIGIBLE
       targetState: INELIGIBLE
+
+# Mitigation journey (03) same session - invalid-dl
+MITIGATION_03_OPTIONS:
+  response:
+    type: page
+    pageId: pyi-driving-licence-no-match-another-way
+    mitigationStart: invalid-dl
+  events:
+    next:
+      targetState: MITIGATION_PP_CRI_UK_PASSPORT
+    end:
+      targetJourney: INELIGIBLE
+      targetState: INELIGIBLE
+
+# Mitigation journey (04) separate session - invalid-dl
+MITIGATION_04:
+  response:
+    type: process
+    lambda: reset-identity
+    lambdaInput:
+      isUserInitiated: false
+      deleteOnlyGPG45VCs: true
+  events:
+    next:
+      targetState: MITIGATION_04_DL_NO_MATCH_PAGE
+
+MITIGATION_04_DL_NO_MATCH_PAGE:
+  response:
+    type: page
+    pageId: pyi-driving-licence-no-match
+  events:
+    next:
+      targetState: MITIGATION_04_IDENTITY_START_PAGE
+
+MITIGATION_04_IDENTITY_START_PAGE:
+  response:
+    type: page
+    pageId: pyi-continue-with-passport
+    mitigationStart: invalid-dl
+  events:
+    next:
+      targetState: MITIGATION_PP_CRI_UK_PASSPORT
+    end:
+      targetJourney: INELIGIBLE
+      targetState: INELIGIBLE
```

Comment:
Same here for `INELIGIBLE_SKIP_MESSAGE`.

Reply:
done